GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -1471,55 +1473,64 @@ private void writeSummaryReport(
         } else {
             createCell(row, 0, sector + " " + REPORT_HEADER, headerStyle);
         }
+
         for (QuestionGroupDto group : orderedGroupList) {
             if (questionMap.get(group) != null) {
                 for (QuestionDto question : questionMap.get(group)) {
-                    if (!(QuestionType.OPTION == question.getType() //TODO: add cascade
-                            || QuestionType.NUMBER == question.getType())) {
+                    boolean doDescriptiveStats = false;
+                    boolean doChart = false;
+                    boolean doDataTable = false;
+                
+                    if (QuestionType.OPTION == question.getType()) { 
+                        doChart = true;
+                        doDataTable = true;                        
+                    } else if (QuestionType.NUMBER == question.getType()) {
+                        doDescriptiveStats = true;
+                        //skip table and chart (phone numbers etc get ridiculous)
+                    } else { //TODO: add cascade
```

Comment:
Is that part of a future issue? Will the cascades be missing?

Reply:
Another issue, yes.